doc/concepts/labels.rst

```diff
@@ -608,6 +618,126 @@ Region expressions
 
       Two regions (left and middle) and their intersection (right).
 
+
+.. _labels-iexpr:
+
+Inhomogeneous Expressions
```

Comment:
Just a thought for all these expression having a `scale` parameter.
Wouldn't it be convenient to create overloads where `scale=1` which
I assume might be the most common case. I might be wrong, though.
Example
```LISP
(distance scale:real loc:locset)
(distance loc:locset) ; same as (distance scale=1 loc:locset)

Reply:
I'm not sure if `scale=1` is going to be a common case. The advantage of always requiring a scale is that it forces the user to think about units, potentially preventing some mistakes. Maybe we could wait and see, if this will be a user request or what scales are common.

Reply:
:tada: Thanks